src/main/resources/db/changelog/changes/03-create-roles-table.yaml

```diff
@@ -17,4 +17,4 @@ databaseChangeLog:
                   type: enum('USER', 'ADMIN')
                   constraints:
                     nullable: false
-                    unique: true
\ No newline at end of file
+                    unique: true
```

Comment:
```suggestion
                    unique: true
                    
```

Reply:
I believe GitHub doesn't display empty lines at the end because it already exists